addons/compat_ace_fire/stringtable.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="diw_armor_plates">
+    <Package name="compat_ace_fire">
+        <Key ID="STR_diw_armor_plates_compat_ace_fire_subCategoryCompat">
+            <English>Compatibility</English>

```

Comment:
that entry can be moved over to the main addon if the string is just `Compatibility` for any other compat mod anyways.